src/components/ProposalStatus/TimeDetail.tsx

```diff
@@ -0,0 +1,48 @@
+import { ProposalState } from 'types/types.guilds.d';
+import { TimeDetailProps } from './types';
+import moment, { unix } from 'moment';
+import { useGuildConfig } from 'Modules/Guilds/Hooks/useGuildConfig';
+
+export const TimeDetail: React.FC<TimeDetailProps> = ({
+  endTime,
+  timeDetail,
+  status,
+  guildId,
+}) => {
+  const { data: guildConfig } = useGuildConfig(guildId);
+  const ONE_MINUTE = 60000;
+
+  if (!guildConfig || status === ProposalState.Active) {
+    return (
+      <span title={endTime?.format('MMMM Do, YYYY - h:mm a')}>
+        {timeDetail}
+      </span>
+    );
+  }
+
+  let { timeForExecution }: any = guildConfig;
+  let executionTimeDetail = '';
+
+  timeForExecution = unix(timeForExecution.toNumber());

```

Comment:
I think in general, manipulating the same variable is considered bad practice. Consider changing it to two separate variables. 

`const { timeForExecution }` and
`const timeForExecutionUnix`